resources/js/admin/users/components/SecurityLogsModal.vue

```diff
@@ -191,7 +192,38 @@ export default {
           auxArray[this.capitalizeKey(key)] = this.booleanToString(value);
         }
       }
-      return auxArray;
+
+      auxArraySort  = this.sortModalArray(auxArray);
+
+      return auxArraySort;
```

Comment:
change to `return this.sortModalArray(ausArray)`

Reply:
Solved.